website/src/auth.tsx

```diff
@@ -59,22 +59,26 @@ export const UserProvider = (props: { children: any }) => {
           If this issue persists, wait and try again later.`)
         break
       case CognitoErrorName.CodeExpired:
-        if (
-          confirm(`This confirmation code has expired. Request a new code?`)
-        ) {
-          resetConfirmationCode()
+        let userResponse = confirm(
+          `This confirmation code has expired. Request a new code?`
+        )
+        if (userResponse && userProvidedEmail) {
+          resetConfirmationCode(userProvidedEmail)
+        } else if (userResponse) {
+          alert("We are unable to send a new code at this time.")
```

Comment:
Perhaps we should not alert the user that they can request a new code if they haven't given us their email.